internal/plugins/openshift/v1/init.go

```diff
@@ -65,19 +65,19 @@ var imageSubstitutions = map[string][]substitution{
 	filepath.Join("config", "default", "manager_auth_proxy_patch.yaml"): {
 		{
 			regexp.MustCompile(`gcr.io/kubebuilder/kube-rbac-proxy:[^ \n]+`),
-			"registry.redhat.io/openshift4/ose-kube-rbac-proxy:v" + ocpProductVersion,
+			"registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9:v" + ocpProductVersion,
 		},
 	},
 	filepath.Join("Dockerfile"): {
 		// Ansible
 		{
 			regexp.MustCompile(`quay.io/operator-framework/ansible-operator:[^ \n]+`),
-			"registry.redhat.io/openshift4/ose-ansible-operator:v" + ocpProductVersion,
+			"registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v" + ocpProductVersion,
```

Comment:
```
[root@preserve-olm-agent-test ~]# podman pull registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v4.16
Trying to pull registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v4.16...
Error: initializing source docker://registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v4.16: reading manifest v4.16 in registry.redhat.io/openshift4/ose-ansible-rhel9-operator: manifest unknown

```

Reply:
The image will be ready after https://github.com/openshift-eng/ocp-build-data/pull/5665 is merged.

Reply:
Thanks~